boards/arc/emsdp/emsdp_em9d.dts

```diff
@@ -51,5 +59,12 @@
 			interrupts = <85 1>;
 		};
 
+		spi@f0008000 {
+			interrupts = <83 1>;
```

Comment:
I would expect the same IRQ line to be used, thus it needs to go to `dts/arc/synopsys/emsdp.dtsi`.

Reply:
em7d_esp has different IRQ line.